packages/app/src/cli/services/dev/output.ts

```diff
@@ -122,9 +122,13 @@ function checkoutUIMessage(url: string, extension: UIExtension) {
   return output.content`Preview link: ${publicURL}`
 }
 
-function customerAccountsUIMessage(url: string, extension: UIExtension) {
-  const publicURL = `${url}/extensions/${extension.devUUID}`
-  return output.content`Preview link: ${publicURL}`
+function customerAccountsUIMessage(storeFqdn: string, url: string, extension: UIExtension) {
+  const [storeName, ...storeDomainParts] = storeFqdn.split('.')
+  const accountsUrl = `${storeName}.account.${storeDomainParts.join('.')}`
+  const host = encodeURIComponent(`${url}/extensions`)
+  const publicURL = `https://${accountsUrl}/extensions-development?host=${host}&extensionId=${extension.devUUID}`
```

Comment:
Do we wanna keep host or switch to something that's technically correct? [Origin](https://developer.mozilla.org/en-US/docs/Glossary/Origin) maybe? 😅

Reply:
`origin` sounds good :thu